appveyor.yml

```diff
@@ -0,0 +1,26 @@
+version: 1.0.0.{build}
+
+platform: x64
+
+branches:
+  only:
+    - master
+
+clone_folder: c:\gopath\src\github.com\sergi\go-diff
+
+environment:
+  GOPATH: c:\gopath
+
+install:
+  - echo %PATH%
+  - echo %GOPATH%
+  - set PATH=%GOPATH%\bin;c:\go\bin;%PATH%
+  - go version
+  - go env
+
+build_script:
+  - cd c:\gopath\src\github.com\sergi\go-diff\
+  - C:\MinGW\bin\mingw32-make install-tools
+  - C:\MinGW\bin\mingw32-make install-dependencies
+  - C:\MinGW\bin\mingw32-make test
```

Comment:
Let's through in a `C:\MinGW\bin\mingw32-make install` before `test` too, I know that `go test` auto builds the packages, but I had builds where `install` cough problems on its own.